tests/test_dxl_single_chain.py

```diff
@@ -1,6 +1,12 @@
 from robot_player import MotionManager, DxlOptions
+from robot_player.dxl.dxl_control_table import DXLPRO, MX106, MX106_P1, MX28, MX28_P1, XSERIES
 import numpy as np
 import platform
+import time
+import csv
```

Comment:
Shouldn't leave imports for other things in test files if possible since other people using your code from different environments might not have those installed and you don't want to introduce extra dependencies for them. You can have another file that does this for your own experiments, just make a separate file.